FormalConjectures/ForMathlib/AlgebraicGeometry/ProjectiveSpace.lean

```diff
@@ -0,0 +1,39 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+import Mathlib.AlgebraicGeometry.Limits
+import Mathlib.AlgebraicGeometry.ProjectiveSpectrum.Scheme
+import Mathlib.RingTheory.MvPolynomial.Homogeneous
+
```

Comment:
The contents of this file will be in mathlib soon, #26061, so maybe let's add a remark about that here so we can adress this once mathlib is bumped. E.g.
```suggestion
import Mathlib.RingTheory.MvPolynomial.Homogeneous

-- The contents of this file will be in mathlib as of #26061 which should be merged in 4.21 or 4.22.

```